sfm_pc/forms.py

```diff
@@ -411,6 +411,8 @@ def save(self, commit=True):
 
         self.instance.object_ref_saved()
 
+        return self.instance
```

Comment:
Previously, we hadn't been returning the object instance created by `form.save()` for either `BaseEditForm` or `BaseCreateForm`. Not only is this bad practice, as it differs from the `save()` spec set out by Django, but it meant that we couldn't access the `self.object` instance in either `get_success_url` method.

Reply:
nice catch!